settings.gradle

```diff
@@ -35,9 +35,14 @@ gradle.ext.openwhisk = [
 
 gradle.ext.scala = [
     version: '2.12.7',
```

Comment:
The core repo has `version: 2.12.10`. Should we align the downstream with that as well?